lib/beacon_web/controllers/media_library_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule BeaconWeb.MediaLibraryController do
+  use BeaconWeb, :controller
+
+  alias Beacon.MediaLibrary
+  alias Beacon.MediaLibrary.Asset
+
+  def show(conn, %{"asset" => asset}) do
```

Comment:
The lookup key `asset` comes from the component `name` attr: `<BeaconWeb.Components.image beacon_attrs={@beacon_attrs} name="dockyard_1.png" />`. That means this name has to be unique, which sounds okay.